test/integration/targets/git/tasks/main.yml

```
@@ -32,7 +32,7 @@
   when:
     - not gpg_version.stderr
     - gpg_version.stdout
-    - git_version.stdout is version("2.1.0", '>=')
+    - not (ansible_os_family == 'RedHat' and ansible_distribution_major_version|int < 7)
```

Use the `version` test here.
```suggestion
    - not (ansible_os_family == 'RedHat' and ansible_distribution_major_version is version('7', '<')
```